market/connector/google-translate/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "google-translate-connector",
+    "name": "Google Translate",
+    "description": "Multi-lingual user interfaces provided by google translate",
```

Comment:
feel free to suggest a better sentence here @andreasbalsiger : this one will be shown on the summary of the market-product.

Reply:
This text is not what the connector is, isn't it? with the connector, you could make multi-lingual user interfaces but the connector is just the API... or am I wrong? it would work, if the included demo contains an example... Can someone help me here?